from wilson.proto.camera_pb2 import Camera
from wilson.proto.color_pb2 import Color
from wilson.proto.colormap_pb2 import ColorMap
from wilson.proto.graph_pb2 import Graph
from wilson.proto.interpolation_pb2 import Interpolation
from wilson.proto.line_pb2 import Line
from wilson.proto.meta_pb2 import ProjectMeta
from wilson.proto.path_pb2 import Path
from wilson.proto.project_pb2 import Project
from wilson.proto.properties_pb2 import ScalarProperty, VectorProperty, ColorProperty
from wilson.proto.sphere_pb2 import Sphere
from wilson.proto.overlay_pb2 import Overlay, TextPosition
from wilson.proto.tube_pb2 import Tube
from wilson.proto.vector_pb2 import Vector
